src/main/java/org/wso2/carbon/inbound/iso8583/listening/ConnectionRequestHandler.java

```diff
@@ -66,65 +62,18 @@ public ConnectionRequestHandler(Socket connection, InboundProcessorParams params
     public void connect() throws IOException {
         if (connection.isConnected()) {
             try {
-                String fromClient;
-                /*
-                jpos sever supports the ISO message with the HeaderLength 0,2 and 4.
-                If the headerLength is 2 or 4, get the ISO message Length from the header value
-                and then read ISO message as byte using that messageLength.
-                Otherwise(headerLength=0) read the ISO message as String.
-                 */
-                if (headerLength != 0) {
-                    int messageLength = getMessageLength(headerLength);
-                    byte[] message = new byte[messageLength];
-                    getMessage(message, 0, messageLength);
-                    fromClient = new String(message);
-                } else {
-                    fromClient = inputStreamReader.readUTF();
-                }
-                ISOMsg isoMessage = unpackRequest(fromClient);
-                isoMessage.setHeader(header);
+                Thread.sleep(100);
```

Comment:
Why sleep? Why 100?

Reply:
It will take time to get the request message from the client in the input stream buffer.